docs/user-event/intro.mdx

```diff
@@ -59,6 +59,8 @@ itself - e.g. in a `before`/`after` hook - for reasons described in
 ["Avoid Nesting When You're Testing"](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing).
 
 ```js
+import userEvent from '@testing-library/user-event'
```

Comment:
I agree with the comments.
On a similar note, I think adding imports to `screen` and `MyComponent` can also be included.

Reply:
Added imports to render and screen, but unsure what to do about MyComponent - I'm not a React person :)

Reply:
I'm not sure we should include the `render` import here, as it (re)introduces references to specific framework libraries.
A code comment might be better:
```jsx
  return {
    user: userEvent.setup(),
    // Import `render` from the framework library of your choice.
    // See https://testing-library.com/docs/dom-testing-library/install#wrappers
    ...render(jsx),
  }
```

Reply:
If we do that for both `screen` and `render` we then end up with almost more comments than code? :)

Should we just have the import for `userEvent` since it is what this page is about?

Reply:
> then end up with almost more comments than code? :)

Not necessarily a bad thing – especially in docs :grin:

Reply:
Just added the suggested comments :)